src/studio/nuxeo-studio/working-in-studio/branding.md

```diff
@@ -391,168 +392,95 @@ history:
 
-1.  Widget Editor
+For their login page background image, most of our customers choose either:
 
-</th><th colspan="1">2\. CSS Editor</th><th colspan="1">3\. Result</th></tr><tr><td colspan="1">
+-   A picture covering all the screen. This is the strategy chosen by Nuxeo.
+-   A logo with a transparent background and a background color.
 
-![]({{file name='blue_background_config.png'}} ?w=300,border=true,thumbnail=true)
-
-</td><td colspan="1">
-
-![]({{file name='-blue_background_style.png'}} ?w=300,border=true,thumbnail=true)
-
-</td><td colspan="1">![]({{file name='blue_background_container.png'}} ?w=300,border=true,thumbnail=true)</td></tr><tr><td colspan="1">
-
-Define a class in the widget editor.
-
-</td><td colspan="1">Declare your blue background in the CSS Editor.
-
-</td><td colspan="1">Enjoy your blue container.</td></tr></tbody></table></div>
+{{#> callout type='warning'}} While selecting your background, be careful not to use heavy files to ensure a quick display to your users.
+{{/callout}}
 
-### Overriding an Existing Rule
+### Selecting a Background Image  
 
-You want to change an existing style to fit your needs? In addition to the [Nuxeo UI Style Guide](http://showcase.nuxeo.com/nuxeo/styleGuide/) that explains the default set of CSS existing rules, you will need to use a browser's web inspector to understand what to override.
+1.  In `Background Image`, click on **Select Resources**.
+1.  Define how the image should be displayed using pixels or the position attribute.
+1.  Indicate if the background covers all the screen (with CSS3 / recent browsers).
+1.  Pick a color that will be displayed if the background image is too small and not covering all the screen.
 
-Let's display the main title of pages in pink:
+|  Before | Studio Configuration | After |
+|---|---|---|
+| ![]({{file name='branding_picture_nuxeo.png'}} ?w=300,border=true) | ![]({{file name='login_customization.png'}} ?w=300,border=true)  |![]({{file name='branding_picture.png'}} ?w=300,border=true)  |
 
-<div class="table-scroll"><table class="hover"><tbody><tr><th colspan="1">1\. Inspect the existing style</th><th colspan="1">2\. Declare your new rule</th><th colspan="1">3\. See the result</th></tr><tr><td colspan="1">![]({{file name='override_inspect.png'}} ?w=300,border=true,thumbnail=true)</td><td colspan="1">![]({{file name='disable_branding.png'}} ?w=300,border=true,thumbnail=true)</td><td colspan="1">![]({{file name='override_result.png'}} ?w=300,border=true,thumbnail=true)</td></tr><tr><td colspan="1">
+### Selecting a Background Video
 
-The pink arrows show the main title at the top of the page, the code and its CSS declaration are at the bottom, in the web inspector. Seeing this, you will need to override the `default.CSS****` `h1` rule that contains the color declaration.
+1.  Select a background video.
+1.  In `Background Video URLs / MIME Type`, click **Add**.
+1.  Enter the URL and MIME Type of your video.
```

Comment:
URLs and MIME types (plural), because depending on the browser some different formats may be supported or not. That could be worth a quick explanation.